crates/humanode-runtime/src/lib.rs

```diff
@@ -788,6 +788,28 @@ impl pallet_balanced_currency_swap_bridges_initializer::Config for Runtime {
     type WeightInfo = ();
 }
 
+pub struct Erc20EvmBalancesMetadata;
+
+impl pallet_erc20_support::Metadata for Erc20EvmBalancesMetadata {
+    fn name() -> &'static str {
+        "Wrapped eHMND"
+    }
+
+    fn symbol() -> &'static str {
+        "WeHMND"
+    }
+
+    fn decimals() -> u8 {
+        18
+    }
+}
+
+impl pallet_erc20_support::Config for Runtime {
```

Comment:
This should be instanced since it could be used with multiple currencies in theory